src/main/java/command/Command.java

```diff
@@ -0,0 +1,27 @@
+package command;
+import exception.MikeCommandExecutionException;
+import mike.TaskList;
+
+public interface Command {
```

Comment:
Maybe you can explain that the command interface interacts with the input of the user and how the classes that implement this interface would have what methods in common.